client/public/index.html

```diff
@@ -11,11 +11,11 @@
     <script
       src='https://api.mapbox.com/mapbox-gl-js/v2.8.2/mapbox-gl.js'></script>
     <link rel='icon' href='<%= BASE_URL %>favicon.ico'>
-    <title><%= htmlWebpackPlugin.options.title %></title>
+    <title>LeadOut</title>
```

Comment:
Do you want to move this to assets/messages/common_messages.ts so we can start moving text out?

Reply:
That's a good idea, but I don't know how to do that. I didn't see anything obvious in [the docs](https://cli.vuejs.org/guide/html-and-static-assets.html#the-index-file).

Reply:
There doesn't seem to be a good way to dynamically set this title ([options](https://stackoverflow.com/questions/62023604/where-to-find-or-how-to-set-htmlwebpackplugin-options-title-in-project-created-w)). It only appears for a moment before router.ts sets it with a better value, so I think any more work here is over-optimization.

But I moved the rest of the titles out into a messages file.